### ISSUE TITLE
Branding and Marketing

### ISSUE THREAD

**Issue by gitjeff2** [hsoft/collapseos#89]
I realize, @hsoft, that you have your own reasons for creating this project. I won't disparage them. This is a *very* cool project for a lot of different reasons. I just want to say that there are several groups who would likely find this project useful (or at least laudable) that have nothing to do with any actual large scale disaster scenarios.

You might want to think about these groups and factor their needs and wants into your roadmap. I suspect you'll see help and useful feedback from people and groups you don't expect. Off the top of my head, I think the following groups could use or get behind Collapse OS each for different reasons. I don't think this is a complete list...

1. **The "Green" Tech Crowd.** Lots of e-waste never gets reused, at least not in the developed world. People in less developed countries who have access to computers either use a *very old* box far past its prime with an equally old version of Windows, or they throw a slimmed down version of Linux on it. But, there are practical limits to *just how tiny* you can make a modern Linux box and how far back in technological time you can go and still have a useful machine. Anything that can be used to write term papers, send email, or print simple documents, will find use in far flung places.
2. **The Retro Computing Crowd.** That's my primary interest. I think a lot of people who love to preserve old tech would take some solace in the fact that they can take several 80's-era  motherboards that are broken beyond repair and cobble together a useful working computer. Also, we're all a bunch of tinkerers and lots of good code could come from this crowd. There are several "new-old" computers being built that might make handy targets: the [Mega65](https://mega65.org/) and the [Commander X16](https://www.pagetable.com/?p=1373) both heavily inspired by the C64 and VIC-20.
3. **The "I Want to Learn Computer Science" Crowd.** I know you're working from a punch list of CPUs you want to get a working kernel on, but you might consider adding a purely [TTL Computer](https://hackaday.com/2016/02/24/8-bit-computer-made-solely-from-nand-gates/) to the list as a potential target. Think about it: you give a 14 year old a soldering iron and some old chips and he not only builds his own CPU from scratch but boots a working OS &mdash; your OS. That would be pretty game changing and motivating for the kid psychologically.
4. **Writers.** There are *many* writers who swear by old products like the [AlphaSmart Dana](https://www.osnews.com/story/131180/the-alphasmart-dana-in-2019/) which is basically just a keyboard with an LED screen and a USB port which mimics a PC keyboard to transfer data. I think this is one of the more practical first world non-disaster use cases for this project. Schools use these things too, in order to teach young children PC keyboard skills on disposable hardware.
5. **Hams.** The Hams that are into volunteer disaster relief might really like this, too. Who cares if a minimalist PC made of spare parts gets swept away or beaten up in a disaster area.

**Comment by hsoft** [comment]
I'm not sure I understand the motivation behind this message. You mean that if Collapse OS was rebranded to be less collapse-y, it would reach a wider audience? Collapse OS is as much a (neo-luddite) manifesto as it's an OS. Without this manifesto, it loses its purpose.

Reaching a wide audience at this juncture is not a priority for the purpose of the project. Wide dissemination of Collapse OS becomes a priority only when the project is finished and that a collapse of internet infrastructure is imminent.

However, I'm sympathetic to anyone who would like to repurpose the code. I'm kinda proud of it. If a fork brings an interesting twist, I will be delighted to link to it. As I write in "CONTRIBUTING.md":

> If you think that Collapse OS could be repurposed, please fork it. If it serves the general purpose of post-collapse resilience, that project could be part of a web of resilience which we'll end up building.

**Comment by hsoft** [comment]
If it's just about widening the scope of the project to cater to a wider audience, then my answer is a bit terser: tightening the scope of the project as much as possible is a necessity. If you don't, complexity creeps in and that has a cost in machine resources. I don't see any other option than forks.

**Comment by gitjeff2** [comment]
> I'm not sure I understand the motivation behind this message. You mean that if Collapse OS was rebranded to be less collapse-y, it would reach a wider audience? 

Your openness about your disaster preparedness motivation earned you a lot of press fairly quickly. So kudos to you. But, yes, that's what I was saying but only after you'd reached your primary goal.

> Collapse OS is as much a (neo-luddite) manifesto as it's an OS. Without this manifesto, it loses its purpose.

Ah. That I didn't get. I follow now. 

All I was saying was that I thought it might have other good uses and not to neglect those possibilities as the OS matures. It sounds like you knew that already and really only care about your specific use case. That's totally fair.

But yes, if you make it plain that you'll link to interesting forks &mdash; and it sounds like you've already said as much and I just missed it &mdash; I think you'll see a lot of interesting parallel developments that might help your project in the long run.

I may buy an [RC2014](https://rc2014.co.uk/ just to play around with it on real hardware.
